apps/challenges/views.py

```diff
@@ -424,6 +424,27 @@ def get_challenge_by_pk(request, pk):
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
 
+@api_view(["GET"])
+@throttle_classes([AnonRateThrottle])
+def get_challenge_by_title(request, title):
```

Comment:
`get_challenge_by_title `-> `get_challenges_by_title`

Reply:
`title` -> `challenge_title`

Reply:
Refactored variable names in f624c2e